linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/JSGen.scala

```diff
@@ -385,7 +385,80 @@ private[emitter] final class JSGen(val semantics: Semantics,
   def genClassDataOf(className: ClassName)(implicit pos: Position): Tree =
     genClassDataOf(ClassRef(className))
 
-  def envModuleField(module: String)(implicit pos: Position): VarRef = {
+  // Compiler generated identifiers --------------------------------------------
+
+  /* We distinguish three types of compiler generated identifiers:
+   *
+   * - envVar: Things pertaining to a given class. Accessible from every chunk
```

Comment:
Should this be called `classVar` if it is associated with a class?

Reply:
We currently call `envVar("c")` `classVar` (through `encodeClassVar`). I think `classVar` *is* a better name than `envVar`, but then we definitely need to rename `encodeClassVar` (or remove it).